src/java/fr/paris/lutece/plugins/mydashboard/modules/identity/web/EditIdentityXPage.java

```diff
@@ -102,7 +105,8 @@ public class EditIdentityXPage extends MVCApplication
     
     private static final String PROPERTY_REDIRECT_MODIFY_ACCOUNT_PAGE         = AppPropertiesService.getProperty( "mydashboard.identity.suspicious.modify_account.redirect.page" );
     private static final String PROPERTY_REDIRECT_MODIFY_ACCOUNT_VIEW         = AppPropertiesService.getProperty( "mydashboard.identity.suspicious.modify_account.redirect.view" );
-
+    private static final String PROPERTY_USER_COMPLETE_ROLE                   = "mydashboard.identity.roleUserComplete";

```

Comment:
this property should have a name more explicit, like 
PROPERTY_ROLE_MYDASHBOARD_REQUIREMENTS_FULFILLED_NAME